article/app/views/liveblog/liveBlogBlocks.scala.html

```diff
@@ -6,10 +6,12 @@
 
 @(blocks: Seq[BodyBlock], article: Article, timezone: DateTimeZone, amp: Boolean = false)(implicit request: RequestHeader)
 
+@import conf.Configuration
+
 @blocks.map { block =>
     <div id="block-@block.id" @if(amp && LiveUpdateAmpSwitch.isSwitchedOn){ data-sort-time="@block.publishedCreatedTimestamp()" } class="block block--content@block.eventClass" itemprop="liveBlogUpdate" itemscope="" itemtype="http://schema.org/BlogPosting">
         <p class="block-time published-time">
-            <a href="/@article.metadata.id?page=with:block-@block.id#block-@block.id" itemprop="url" class="block-time__link">
+            <a href="@if(amp){@Configuration.site.host}/@article.metadata.id?page=with:block-@block.id#block-@block.id" itemprop="url" class="block-time__link">
```

Comment:
What do people think of this? I'm making the permalink in the timestamp an absolute URL in AMP. Should this be an absolute URL in non-AMP also?